etc/ordered_pip.sh

```diff
@@ -4,7 +4,7 @@ a=0
 while read line
 do     
     if [[ -n "$line" && "$line" != \#* ]] ; then
-        pip install $line
+        pip3 install $line
```

Comment:
@twiecki done. seems like the branch has drifted from master a bit. Should I rebase?